examples/opencv/README.md

```diff
@@ -0,0 +1,14 @@
+# Distributed Image Processing with OpenCV and Python Pachyderm
```

Comment:
Do you think we should move this to Read The Docs as well? Or at least link this to our docs page?

Reply:
The docs page has a link to the `examples/` folder. I'm okay with the way it is because right now we have two simple examples on the "Getting Started" page. Adding this one as it is might not be as useful since the readme just redirects them to the core pach opencv example to understand what's going on. We could walk through the opencv example step by step ourselves, but that seems redundant to what we already in core pach. The python file has comments too which makes it pretty straightforward to understand too.